### ISSUE TITLE
Pages and spreads

### ISSUE THREAD

**Issue by mrdanielamara** [Simbul/baker#31]
Hello,

I would really like to see a way to switch between pages and spreads. This is inherent to book structure and design. For those of us who design books, we design in spreads, and it would be nice to design in this new format using the same standards. Specifically, I'd like to see a switch from page to spreads on rotating the ipad to the horizontal. I believe Adobe's new digital magazine standard does this by keeping two sets of files, one for vertical layout and one for horizontal. I don't know if that is the best solution, but it would work. Maybe a simpler solution would be to logically placing two pages next two each other in same order a book would (spreads). So if you are looking at page 2, and rotate, you should get the spread containing pages 2 and 3; and the same should occur if you were on page 3, because it is part of the page 2 and 3 spread. 

Hope that makes sense. I really can't image designing a book without spreads. And simply reorienting everything with media query is not enough. 

Thanks

Daniel

**Comment by folletto** [comment]
Could you try to detail a little more how it could work?
Because I can't think right now of an easy way of doing that.

Do you also have an example? Because from your explanation I'm imagining a "spread" with two pages really cramped trying to fit 768px inside two 512px columns and I don't understand the need for "spread" that couldn't just be solved developing your book horizontally. :)

Thanks!

**Comment by mrdanielamara** [comment]
Sure. Take a look at this youtube video showing some of Adobe's digital publish platform, horizontal possibilities.  http://www.youtube.com/watch?v=MHvaKv-LXEw&feature=related
You can see sometimes they just change layout, and sometimes they show full spreads. Which is a benefit to having two sets of files. 

I think if you go open any designed book (from photography, graphic design, popular culture) you'll see plenty of pages where images, illustrations, diagrams, and even text fall across a full spread. The effect just isn't the same for a single page, but I wouldn't want to force people into a horizontal reading either. You might try looking at some book from the publisher Chronicle Books. 

I agree, it would compress text very small, but these types of books, are generally lighter on text. And zoom could be accomplished simply by changing orientation back to vertical. 

I really like what you've made. And I hope to avoid Adobe Draconian system, in favor of an open one, where I can code and design as I would like.

If you'd like I could probably code up a quick example. And many thanks for the ultra fast response.

**Comment by folletto** [comment]
Don't you know any currently released iPad app that does that? Because from the video it looks like just a change of the CSS keeping the same content, I'd like to see how it works to get it correctly. :)

**Comment by mrdanielamara** [comment]
Sorry I've been MIA. Holidays.

Here is a link to the Vanity Fair magazine video. Go about half way through, and you can see their spread view. 
http://www.youtube.com/watch?v=JVmWkM6uXDY&feature=related

Also, take a look at these videos of the Adobe digital publish platform. This is how many of these magazines are being made.
http://tv.adobe.com/show/digital-publishing/

**Comment by folletto** [comment]
From all the videos you linked, and from the tutorials and docs of the Adobe platform, it seems that they _don't_ merge pages to create spreads. :)

I've scouted around a bit, because it felt a bit strange from a cognitive point of view, and I found:
http://download.macromedia.com/pub/labs/digitalpublishing/digitalpublishing_userguide.pdf

On page 21 it says:

> "You cannot mix single-orientation and dual-orientation stacks in an issue."

That means also that you must have also exactly the same number of pages. In fact, Adobe works in stacks (articles), and if you see, Vanity Fair's magazine when you move in spread view doesn't change the number of page - no merged spreads - but adds an advertisement on the side.

I've tested also a few other apps developed on the Adobe platform and the page count is always the same regardless of orientation.

If I may suggest the reason, that's quite understandable: I could infer that if while browsing a digital magazine it changes the page numbers "live" when I turn the iPad, then I would be confused, because the working unit is a single article (stack in Adobe's dictionary) and so I wouldn't know anymore where the article is.

So, I'd say that the best solution using Baker is just keeping all the content in the same HTML, and using CSS to show/hide/rearrange elements on the page when the user change the orientation. :)
